# Is the gripper closed
bool closed
---
